most-adapter/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cycle/most-adapter",
-  "version": "3.0.0",
+  "version": "3.1.0",
```

Comment:
You don't need to bump this version. It's done automatically by our toolchain. For the time being I'll just merge this and revert the bump, so just letting you know this isn't needed the next time.